container/push.bzl

```diff
@@ -148,10 +140,10 @@ container_push = rule(
                 "OCI",
                 "Docker",
             ],
-            doc = "The form to push: Docker or OCI.",
+            doc = "The form to push: Docker or OCI, default to 'Docker'.",
         ),
         "image": attr.label(
-            allow_single_file = [".tar"],
+            allow_files = True,
```

Comment:
why change from allows_single_file to allow_files?

Reply:
Fixed. I didn't explicitly change this. I just renamed new_push.bzl to push.bzl.